WebAPICore/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,33 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace WebAPICore.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class WeatherForecastController : ControllerBase
```

Comment:
remove autogenerated controller if not required

Reply:
auto generated Controller deleted